### PR TITLE
Explorer Metaplex NFTs: Adding External URL Button 

### PR DESCRIPTION
**Pull request by RTISCOOL**
Edit: Originally added a big ugly button but now adds a field to the NFT overview section

Proposes an addition of a button/link to the metaplex NFT's external url on the explorer, tokens already have this property implemented in the explorer but nfts do not. The external_url property is already supplied in the metaplex URI JSON Schema and some NFT's have already implemented it. Adding this button will allow NFT creators to easily link back to their marketplace, or any other representation of the asset that cant otherwise be shown on the explorer. 

#### Summary of Changes
- Moved the logic from useExtendedArt(fetching json from URI) from the NFTArt component to the account provider allowing the URI JSON content to be loaded prior to rendering helps with this request as well as others such as issue #21109 
- Added a Section and link to display the external_url property in the NFT overview section

#### NFTs used to test
(grabbed 2 random nfts off a marketplace) 

explorer_url set in metadata: c8BeYNNFJhtgXYryfAgx97Y3qD4xLWNtuuhuuG6s4Hj
Screen Shot 2021-11-02 at 1 30 22 PM

No explorer_url set in metadata: DDWchxJ5Yqi5Vzz7K6RqoKS1T77zurfX1hFnzCDYjo4h
Screen Shot 2021-11-02 at 1 29 51 PM

Thanks guys!

### CODE REVIEW

**Comment by VertCode** [comment]
Would love to see this added, looks great and will use this very often when its added.

**Comment by roederw** [comment]
I don't have much of a say here, but I am not in favor of adding this change. Unless we allow other off-chain redirection (would love to see examples if there already is!) @oJshua @jstarry for visibility.

**Comment by roederw** [comment]
From a visual standpoint, I think we could do a bit better. Maybe have a static title for the redirection element? Potentially something similar to non-NFT Tokens? See the website field [here](https://explorer.solana.com/address/7xKXtg2CW87d97TXJSDpbD5jBkheTqA83TZRuJosgAsU).

Also, moving the `extendedArt` logic might not be the right call.

**Comment by RTISCOOL** [comment]
> From a visual standpoint, I think we could do a bit better. Maybe have a static title for the redirection element? Potentially something similar to non-NFT Tokens? See the website field [here]. That way you can just add the logic to `NFTSection` (https://explorer.solana.com/address/7xKXtg2CW87d97TXJSDpbD5jBkheTqA83TZRuJosgAsU).
> 
> Also, moving the `extendedArt` logic might not be the right call.

I agree with this, would match the way regular spl tokens are displayed. I'll get to work on this in a sec. 

As for the extendedArt logic, do you think it would be a better idea to move it directly into the fetchAccountInfo provider? Theres a lot of info on that URI which could potentially need to be used in the initially on the account page (looking at issue #21109 here)

File: explorer/src/scss/_solana.scss [hunk 1]
```diff
@@ -406,6 +406,11 @@ p.updated-time {
   width: 150px;
 }
 
+.external-url-button {
```
Comment (jstarry) [inline explorer/src/scss/_solana.scss:409]: nit: remove please

File: explorer/src/components/account/TokenAccountSection.tsx [hunk 2]
```diff
@@ -355,6 +355,21 @@ function NonFungibleTokenMintAccountCard({
             />
           </td>
         </tr>
+        {nftData?.json && nftData.json.external_url && (
+          <tr>
+            <td>Website</td>
+            <td className="text-lg-right">
```
Comment (jstarry) [inline explorer/src/components/account/TokenAccountSection.tsx:361]: nit: rebase off master and change to `text-lg-end`

File: explorer/src/components/account/TokenAccountSection.tsx [hunk 3]
```diff
@@ -355,6 +355,21 @@ function NonFungibleTokenMintAccountCard({
...
+              <a
+                rel="noopener noreferrer"
+                target="_blank"
+                href={nftData.json.external_url}
+              >
+                {nftData.json.external_url}
+                <span className="fe fe-external-link ml-2"></span>
```
Comment (jstarry) [inline explorer/src/components/account/TokenAccountSection.tsx:368]: nit: rebase off master and change to `ms-2`

**Review by jstarry — COMMENTED** [review]
Looks great! Thanks for your patience on the review

**Comment by jstarry** [comment]
Ugh sorry I shouldn't have merged the metaplex update before your change. Do you mind fixing the conflicts? If not, I can take care of them

**Comment by RTISCOOL** [comment]
> Ugh sorry I shouldn't have merged the metaplex update before your change. Do you mind fixing the conflicts? If not, I can take care of them

Ill take care of it real quick

**Comment by jstarry** [comment]
Thanks again!

**Pull request merged**